python/cudf_polars/tests/expressions/test_stringfunction.py

```diff
@@ -39,3 +41,70 @@ def test_unsupported_stringfunction():
 
     with pytest.raises(NotImplementedError):
         _ = translate_ir(q._ldf.visit())
+
+
+def test_contains_re_non_strict_raises():
+    df = pl.LazyFrame({"a": ["a"]})
+
+    q = df.select(pl.col("a").str.contains(".", strict=False))
+
+    with pytest.raises(NotImplementedError):
+        _ = translate_ir(q._ldf.visit())
+
+
+def test_contains_re_non_literal_raises():
+    df = pl.LazyFrame({"a": ["a", "b"], "b": [".", "e"]})
+
+    q = df.select(pl.col("a").str.contains(pl.col("b"), literal=False))
+
+    with pytest.raises(NotImplementedError):
+        _ = translate_ir(q._ldf.visit())
+
+
+@pytest.fixture
```

Comment:
We might prefer to put the fixture at the top. Also, since this fixture and the data used in some of the other tests all conveniently expect a column named `a` we might be able to get away with a single fixture for this whole file for now.

Reply:
Done.